content/millennium/r4/financial/coverage.md

```diff
@@ -82,6 +82,19 @@ _Implementation Notes_
 
 <%= disclaimer %>
 
+#### Example - Patient-level Public Healthcare Coverage
+
+#### Request
+
+    GET https://fhir-ehr-code.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/Coverage?patient=12724066
```

Comment:
Why is the patient exactly the same as the Patient-level Coverage? While the responses are different. Might be confusing to users

Reply:
So this patient returns both private and public healthcares I believe.. in the responses we are just showing one coverage or the other for clarity.. I'm not sure if we will be able to find a patient with just a private healthcare in C1941.. if you think it it is truly confusing, we could create a new patient with just a private coverage..

But in conclusion, this search will return a bundle of multiple coverages with both private and public healthcares, we only highlight one to show the differences.

Reply:
I think it's probably fine, since we already added a disclaimer that it may not return the exact data.